Configurations.md

```diff
@@ -1265,6 +1265,31 @@ Maximum width of each line
 
 See also [`error_on_line_overflow`](#error_on_line_overflow).
 
+## `merge_derives`
+
+Merge multiple derives into a single one.
+
+- **Default value**: `true`
+- **Possible values**: `true`, `false`
+
+*Note*: The merged derives will be put after all other attributes or doc comments.
```

Comment:
Thank you for the feedback! I will update this PR so that only consecutive derives will get merged to preserve ordering.